types/hapi/v16/index.d.ts

```diff
@@ -101,7 +105,7 @@ export class Server extends Podium {
      * An object providing access to the server methods cs://hapijs.com/api/16.1.1#servermethodname-method-options} where each server method name is an object property.
      * [See docs](https://hapijs.com/api/16.1.1#servermethods)
      */
-    methods: Dictionary<ServerMethod>;
+    methods: ServerMethods & Dictionary<ServerMethod>;
```

Comment:
It seems like this intersection is redundant, and the other versions you edited just use `ServerMethods`—is this intentional?

Reply:
Mistake, fixed.